zerver/migrations/0001_initial.py

```diff
@@ -47,8 +53,8 @@ class Migration(migrations.Migration):
 CREATE TRIGGER zerver_message_update_search_tsvector_async
   BEFORE INSERT OR UPDATE OF subject, rendered_content ON zerver_message
   FOR EACH ROW EXECUTE PROCEDURE append_to_fts_update_log();
-"""),
-            ]
+""")
+        zulip_postgres_migrations = [full_text_search_migration]
 
```

Comment:
Sorry. My description didn't have enough information.

What the script needs to do is what you list. It's not a large task as you said.

I think that we need to decide the followings things. It may be a large task because we may need many things or discussions.

  * Where should we put the script?
  * How do users use the script?